app/src/main/java/com/hemdan/mvvm/ui/main/actorslist/ActorsListFragment.kt

```diff
@@ -0,0 +1,62 @@
+package com.hemdan.mvvm.ui.main.actorslist
+
+import android.os.Bundle
+import android.view.View
+import androidx.lifecycle.Observer
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.hemdan.mvvm.R
+import com.hemdan.mvvm.data.model.PopularInfo
+import com.hemdan.mvvm.ui.base.BaseFragment
+import kotlinx.android.synthetic.main.fragment_actors_list.*
+import java.util.ArrayList
+import javax.inject.Inject
+
+class ActorsListFragment : BaseFragment() {
+
+    private lateinit var listAdapter: ActorsAdapter
+    private var listLayoutManager: LinearLayoutManager? = null
+
+    private var popularInfos = ArrayList<PopularInfo>()
+
+    override fun getLayoutResourceId(): Int = R.layout.fragment_actors_list
+
+    @Inject
+    lateinit var actorsListViewModel: ActorsListViewModel
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        listLayoutManager = LinearLayoutManager(context)
```

Comment:
@MonaAliMahmoud you can create method called setupList() with all related list functions like 
actorsList.adapter = adaper
actorsList.layoutManager = LinearLayoutManager(context)
actorsList.addOnScrollListener ...etc